src/containers/Routes.jsx

```diff
@@ -92,58 +81,19 @@ const Routes = () => {
         path={getViewEntityPathsList('/dac/:slug')}
         render={({ match }) => <Redirect to={`/community/${match.params.slug}`} />}
       />
-      <Route
-        exact
-        path="/communities/:id/edit"
-        render={props => (
-          <EditCommunity
-            balance={balance}
-            isForeignNetwork={isForeignNetwork}
-            displayForeignNetRequiredWarning={displayForeignNetRequiredWarning}
-            {...props}
-          />
-        )}
-      />
-      <Route
-        exact
-        path="/campaigns/new"
-        render={props => (
-          <EditCampaign
-            isNew
-            balance={balance}
-            isForeignNetwork={isForeignNetwork}
-            displayForeignNetRequiredWarning={displayForeignNetRequiredWarning}
-            {...props}
-          />
-        )}
-      />
+      <Route exact path="/campaigns/new" render={props => <EditCampaign isNew {...props} />} />
```

Comment:
If props are not needed they should be removed from these components too

Reply:
EditCampaign needs match props